test/new_relic/agent/instrumentation/active_record_helper_test.rb

```diff
@@ -62,5 +62,20 @@ def test_product_operation_collection_for_name_with_integer_returns_nil
       assert_equal "other", operation
       assert_nil collection
     end
+
```

Comment:
New tests to confirm that `<adapter name>_makara` adapters work as desired.

Reply:
Hmm... indeed. I think I saw the existence of that dedicated subscriber class in the test directory and just decided to forget I'd ever seen it rather than try to figure out how it worked. Will try to get some updates going.

Reply:
Addressed with [5d6d93b](https://github.com/newrelic/newrelic-ruby-agent/pull/1180/commits/5d6d93b5e64b4d667b57d2734566f95b9c311c5f)